pkg/agent/openflow/pipeline.go

```diff
@@ -2739,9 +2740,25 @@ func (f *featurePodConnectivity) l3FwdFlowToNode() []binding.Flow {
 	return flows
 }
 
-// l3FwdFlowToExternal generates the flow to match the packets destined for external network.
+// l3FwdFlowToExternal generates the flow to forward the packets destined for external network. Corresponding cases are
```

Comment:
```suggestion
// l3FwdFlowToExternal generates the flow to forward the packets destined for the external network. Corresponding cases are
```

Reply:
I think I can change this in a later PR since all tests have been passed now.